### ISSUE TITLE
Provide "Swap" option for colliding items

### ISSUE THREAD

**Issue by 555ea** [swiety85/angular2gridster#105]
Hi!

I'm currently working on Jira-like ticket tracker with sprint tickets being draggable through angular2gridster.  The problem I encountered is that in a block with two rows and several columns, where there are two elements, [x: 4, y: 0] and [x: 4, y: 1], when we transfer the lower  [x: 4, y: 1] element on top of higher [x: 4, y: 0] element, the upper does not take the place of the bottom, but moves to the left, according to some collision resolving strategy.

 I'm looking for a way for them to just swap each with other, just like in angular-gridster.
![screenshot_527](https://user-images.githubusercontent.com/13518952/31016378-133fce46-a52d-11e7-94ad-93b2941512c4.png)

**Comment by swiety85** [comment]
Hi, the behaviour you expect (swap) only can happen when there is no space on sides. Also there is no configuration for that, that would change this strategy.
I'm not sure if I understood you right, but if you want to make a Jira like board (or something like [this](https://github.com/swiety85/angular2gridster/projects/1)), that you want to move tasks from one lane to another, then quite soon I will provide possibility to drag n drop items between dashboards (#28). In this case you will be able to create couple gridsters (as lanes) with items that occupy full width - then swapping will work as expected.

**Comment by 555ea** [comment]
Yes, this is what I am looking for, but I personally think that separate gridsters as lanes are a bit too complicated. Take a look at angular-gridster, they have a swapping option, 

> swapping: false, // whether or not to have items of the same size switch places instead of pushing down if they are the same size
https://github.com/ManifestWebDesign/angular-gridster

You can test it at their demo, by "Enable Draggability" and "Enable Swapping".
https://rawgit.com/ManifestWebDesign/angular-gridster/master/index.html#/main
